sdk/iot/Azure.ResourceManager.IotFirmwareDefense/CHANGELOG.md

```diff
@@ -1,11 +1,16 @@
 # Release History
 
-## 1.1.0-beta.1 (Unreleased)
+## 2.0.0-beta.1 (Unreleased)
```

Comment:
Mgmt package should never bump its major version, we must change it back to `1.1.0-beta.1`